Makefile

```diff
@@ -121,7 +121,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	$(KUSTOMIZE) --load-restrictor LoadRestrictionsNone build config/deploy/local | kubectl apply -f -
```

Comment:
What does `--load-restrictor LoadRestrictionsNone` do?

Reply:
It disables the restriction that the env files have to be under the kustomization directory, so we can reference them from the root of the project

Reply:
Interesting, good to know.